tools/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.10-alpine
+
+LABEL a01.product="azuresdkforgo" \
+    a01.index.schema="v2" \
+    a01.env.AZ_SUBSCRIPTION_ID="secret:sp.subscriptionid" \
+    a01.env.AZ_TENANT_ID="secret:sp.tenant" \
+    a01.env.AZ_CLIENT_ID="secret:sp.clientid" \
+    a01.env.AZ_CLIENT_SECRET="secret:sp.clientsecret" \
+    a01.env.AZ_SP_OBJECT_ID="secret:sp.objectid" \
+    a01.env.AZ_LOCATION="arg-mode"
+
+COPY . /go/src/github.com/Azure-Samples/azure-sdk-for-go-samples
+ADD https://a01tools.blob.core.windows.net/droid/linux/a01droid /app/a01droid
+
+RUN apk add --no-cache git bash && \
+    go get github.com/golang/dep/cmd/dep && \
+    chmod +x /app/a01droid && \
+    mv /go/src/github.com/Azure-Samples/azure-sdk-for-go-samples/tools/get_index /app/get_index && \
+    go run /go/src/github.com/Azure-Samples/azure-sdk-for-go-samples/tools/list/list.go > /app/test_index
+
+
+WORKDIR /go/src/github.com/Azure-Samples/azure-sdk-for-go-samples
+RUN dep ensure
+
+WORKDIR /
+CMD /app/a01droid
```

Comment:
where is this coming from ?

Reply:
It is part of a01 automation. The droid will initiate a test in the container.
https://github.com/Azure/adx-automation-client/blob/master/onboard.md

Reply:
ok but how does it get into the container described by this dockerfile. Is it from a mount/volume ?

Reply:
I gets downloaded on Dockerfile line 13

Reply:
oh sorry I missed that part. So is that blob exposed to the internet , no auth necessary ?  is it compliant to have a publicly exposed endpoint ?

Reply:
yes i was wondering how do you know from this dockerfile that what is on that random blob is the same thing from the github repo. And regardless of the code being public or not, having an exposed endpoint might raise issues. please check with Troy if this might be a problem.

Reply:
@troydai Am I assuming correctly that the `droid` container has a `blob` access policy?

Reply:
From what I see in the portal, yes, the container has a blob access policy